src/components/Footer.js

```diff
@@ -1,44 +1,63 @@
 import React from "react"
-import Title from "./Title"
-import { graphql, useStaticQuery } from "gatsby"
-import { Box, Text } from "@chakra-ui/core"
-import Img from "gatsby-image"
+import { Box, PseudoBox, Heading } from "@chakra-ui/core"
 
-const Footer = () => {
-    const { organizers } = useStaticQuery(graphql`
-        query {
-            organizers: file(relativePath: { eq: "organizers.jpg" }) {
-                childImageSharp {
-                    fluid(maxWidth: 2000, maxHeight: 700) {
-                        ...GatsbyImageSharpFluid_withWebp
-                    }
-                }
-            }
-        }
-    `)
-    return (
+export default () => (
+    <Box px="10vw" py="75px" color="blue.500">
```

Comment:
don't use vw, it's bad practice for responsiveness, just use charka default breakpints